ViAn/Filehandler/filehandler.cpp

```diff
@@ -8,8 +8,29 @@ FileHandler::FileHandler() {
     this->m_fid = 0;
     this->m_did = 0;
     this->lastError = 0;
+    #ifdef _WIN32
+        this->workSpace = "C:/";
+    #elif __APPLE__
+        this->workSpace = "/Applications/";
+    #elif __unix__
+        this->workSpace = "~/";
+    #endif
+
+    //ID id = add_file("ViAn_config.txt"); Will be used to store current workspace and other run-to-run coonstans
+
+
+
 
 }
+/**
+ * @todo save workspace to file
+ * @brief FileHandler::set_workspace
+ * @param newWorkSpace
+ */
+void FileHandler::set_workspace(std::string newWorkSpace){
```

Comment:
Remember to use snake_case for variable names: new_work_space instead of newWorkSpace